FAULT TYPE: OPENSTACK_THROW_EXCEPTION-VOLUME
TARGET COMPONENT: /cinder/volume/flows/manager/create_volume.py
TARGET CLASS: CreateVolumeFromSpecTask
TARGET FUNCTION DEF: _capture_volume_image_metadata+617;8
FAULT POINT: self.db.volume_glance_metadata_bulk_create(context, volume_id, volume_metadata)
